source/nodejs/adaptivecards/README.md

```diff
@@ -21,6 +21,8 @@ Please be aware of the following **breaking changes** in particular versions.
 | In version | Change description |
 |---|---|
 | **1.2** | The default `value` of an Input.Time **no longer accepts seconds**. 08:25:32 will now be treated as an invalid value and ignored; it should be replaced with 08:25. This behavior is consistent with other Adaptive Card renderers.|
+|| The `ICardObject` interface has been **REMOVED**, replaced with the `CardObject` class that both `CardElement` and `Action` extend. This change should have little to no impact on any application.|
```

Comment:
thanks! should this be in the 1.2 breaking change list as well? If so i'll make a note of it in our 1.2 release Onenote where we are tracking the breaking changes etc

Reply:
It is in 1.2. I purposely not repeat 1.2 in the first column.

Reply:
ha no i meant the 1.2 release notes markdown file.. No matter though; I've noted it as such

---
In reply to: [284927257](https://github.com/microsoft/AdaptiveCards/pull/2905#discussion_r284927257) [](ancestors = 284927257)